### PR TITLE
docs(fs): add docs to the fs module

### PR DESCRIPTION
**Pull request by pevisscher**

### CODE REVIEW

**Comment by pevisscher** [comment]
Current dependencies on/for this PR:
* main
  * **PR #26** <a href="https://app.graphite.dev/github/pr/skyleague/axioms/26" target="_blank">Graphite</a> 
    * **PR #27** <a href="https://app.graphite.dev/github/pr/skyleague/axioms/27" target="_blank">Graphite</a> 
      * **PR #28** <a href="https://app.graphite.dev/github/pr/skyleague/axioms/28" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/skyleague/axioms/28?utm_source=stack-comment).

File: src/fs/fstat/fstat.ts [hunk 1]
```diff
@@ -6,6 +6,28 @@ import { Nothing } from '../../type/maybe'
...
+ * await fstat("./existing")
+ * // => { right: Stats }
+ * ```
+ *
+ *
+ * @param path - The path to normalize
+ * @param options - Configure how the normalization should happen.
+ * @param options.forceForwardSlash - Set to true if we need to replace all backwards slashes to forward. By default
+ *                                    we check the separator used on the current OS, and only replace when needed.
+ * @returns The normalized path.
```
Comment (MickVanDuijn) [inline src/fs/fstat/fstat.ts:26]: copy-paste?

**Review by MickVanDuijn — CHANGES_REQUESTED** [review]
See inline comments

File: src/fs/fstat/fstat.ts [hunk 2]
```diff
@@ -6,6 +6,28 @@ import { Nothing } from '../../type/maybe'
...
+ *
+ * await fstat("./existing")
+ * // => { right: Stats }
+ * ```
+ *
+ *
+ * @param path - The path to normalize
+ * @param options - Configure how the normalization should happen.
+ * @param options.forceForwardSlash - Set to true if we need to replace all backwards slashes to forward. By default
+ *                                    we check the separator used on the current OS, and only replace when needed.
```
Comment (MickVanDuijn) [inline src/fs/fstat/fstat.ts:25]:
```suggestion
```

**Review by MickVanDuijn — CHANGES_REQUESTED** [review]
See inline comments

**Review by MickVanDuijn — APPROVED** [review]

**Comment by pevisscher** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/skyleague/axioms/28?utm_source=stack-comment).


Job Reference: TdjwnjsAVSOrsN07nKI6

**Pull request merged**